### PR TITLE
Include missing references on some checks. 

### PR DESCRIPTION
**Pull request by diegoakechi**
This small PR includes missing references on some checks (before listed as `T.B.D`).
It also fix some typos and complement functional information in some of the checks.

### CODE REVIEW

File: runner/ansible/roles/checks/1.3.1/defaults/main.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ remediation: |
      ```crm configure property maintenance-mode=false```
 
   ## References
-  - T.B.D.
+  - https://documentation.suse.com/sle-ha/15-SP3:/html/SLE-HA-all/cha-ha-storage-protect.html
```
Comment (arbulu89) [inline runner/ansible/roles/checks/1.3.1/defaults/main.yml:31]: This link doesn't work. Maybe some typo?

Comment (arbulu89) [reply runner/ansible/roles/checks/1.3.1/defaults/main.yml:31]: Ok, I see that there is `:` in the URL. Can you fix it @diegoakechi ?

**Review by arbulu89 — CHANGES_REQUESTED** [review]
@diegoakechi There is a typo in 1 of the links

File: runner/ansible/roles/checks/1.3.1/defaults/main.yml (hunk 1 above)
Comment (diegoakechi) [reply runner/ansible/roles/checks/1.3.1/defaults/main.yml]: done

**Review by arbulu89 — APPROVED** [review]

**Pull request merged**